demo/Dockerfile.alpine

```diff
@@ -17,7 +16,7 @@
 #   To run tests, use `--build-arg WITH_TESTS=true`
 #
 #   # Note: In this case, I would suggest to map port `5000` to `5000`. The user can always change the port according to their needs.
-#   docker run -it --rm --init --name agent -p5000:5000 \
+#   docker run -it --rm --init --name agent -p5000:5000 -v ... \
 #     mtconnect/agent:2.0.0.12_RC18
```

Comment:
I suggext to remove the version tag unless you want to update in on every release, unless you want to move the whole notes in the docs (then it should be done there). I suggest to do this in all four dockerfiles.

```suggestion
#     mtconnect/agent
```

Reply:
Agree. I need to do it in the workflow. It will required a little sed script to change the ref name.

Reply:
Changed to `#     --tag mtconnect/agent:latest \`

Reply:
I just wonder if `--tag` is actually needed. :thinking: